src/managers/KeybindManager.cpp

```diff
@@ -949,6 +949,12 @@ void CKeybindManager::moveActiveToWorkspace(std::string args) {
 
     POLDWS->m_pLastFocusedWindow = g_pCompositor->getFirstWindowOnWorkspace(POLDWS->m_iID);
 
+    CMonitor* pOldMonitor = g_pCompositor->getMonitorFromID(POLDWS->m_iMonitorID);
+    if (pWorkspace->m_bIsSpecialWorkspace)
+        pOldMonitor->setSpecialWorkspace(pWorkspace);
```

Comment:
I believe in this case this would be correct, but this should be pMonitor.

ref above, `pMonitor = g_pCompositor->getMonitorFromID(pWorkspace->m_iMonitorID);`

So pMonitor has pWorkspace

no?

Reply:
Yeah indeed. Fixed.